src/modules/product/product.service.ts

```diff
@@ -0,0 +1,24 @@
+import { Injectable } from '@nestjs/common';
+
+import { OrderEntity } from 'src/entities/order.entity';
+import { ProductEntity } from 'src/entities/product.entity';
+import { OrderRepository } from 'src/modules/repository/services/order.repository';
+import { ProductRepository } from 'src/modules/repository/services/product.repository';
+
+@Injectable()
+export class ProductService {
+  constructor(
+    private readonly productRepository: ProductRepository,
+    private readonly orderRepository: OrderRepository,
+  ) {}
+
+  public async getAllProducts(): Promise<ProductEntity[]> {
+    const products = this.productRepository.find();
+    return products;
```

Comment:
const product is redundant, + no await in async

Reply:
maybe add an empty array in return type in case there are no products